global_costmap:
    global_frame: odom
    robot_base_frame: base_link
    plugins:
        - {name: unrolling_layer,       type: "unrolling_layer::UnrollingLayer"}
        - {name: inflation_layer,       type: "costmap_2d::InflationLayer"}
    publish_frequency: 5.0
    footprint: [[0.4,0.4],[-0.4,0.4],[-0.4,-0.4],[0.4,-0.4]]
    width: 200
    height: 200
    origin_x: -100
    origin_y: -100
    track_unknown_space: true
    unknown_cost_value: 30
    rolling_window: false
    resolution: 0.1
    inflation_layer:
        inflation_radius: 5.0
unrolling_layer:
    topic: "/move_base_flex/local_costmap/costmap"
